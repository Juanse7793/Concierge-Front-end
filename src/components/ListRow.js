import React from 'react';
import PropTypes from 'prop-types';

const ListRow = ({ caption, text }) => (
  <li className="row">
    <span>{caption}</span>
    <span>{text}</span>
  </li>
);

ListRow.propTypes = {
  caption: PropTypes.string.isRequired,
  text: PropTypes.string.isRequired,
};

export default ListRow;
