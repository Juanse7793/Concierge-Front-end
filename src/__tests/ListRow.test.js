import renderer from 'react-test-renderer';
import ListRow from '../components/ListRow';

it('matches snapshot', () => {
  const component = renderer.create(
    <ListRow
      caption="caption"
      text="text"
    />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
