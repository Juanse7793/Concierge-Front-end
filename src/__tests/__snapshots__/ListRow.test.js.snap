// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<li
  className="row"
>
  <span>
    caption
  </span>
  <span>
    text
  </span>
</li>
`;
